src/ngx_http_lua_headers_in.c

```diff
@@ -645,6 +645,78 @@ ngx_http_clear_builtin_header(ngx_http_request_t *r,
 }
 
 
+static    ngx_hash_init_t             lua_req_header_hash;
+static    u_char header_string[512];
+ngx_int_t
+lua_req_set_header_hash_init(void)
+{
+    ngx_int_t                   rc;
+    ngx_uint_t                  i;
+    ngx_uint_t                  pos = 0;
+    ngx_uint_t                  key_max_len = 0;
+    ngx_str_t                   tmp_key = {0, &header_string[0]};
+    ngx_hash_keys_arrays_t      ha;
+    ngx_http_lua_set_header_t        *handlers = ngx_http_lua_set_handlers;
```

Comment:
Style: the variable identifiers need to be aligned unless separated by a blank line.